python/lsst/ts/eas/diurnal_timer.py

```diff
@@ -285,6 +352,20 @@ async def stop(self) -> None:
         async with self.twilight_condition:
             self.twilight_condition.notify_all()
 
+        async with self.sunrise_condition:
+            self.sunrise_condition.notify_all()
+
+    def get_twilight_time(self, after: Time) -> Time:
+        """Returns the time of next end of twilight for the given time.
+
+        Parameters
+        ----------
+        after : `~astropy.time.Time`
+            Time returned will be time corresponding to the
```

Comment:
`Time returned will be time corresponding to the end of twilight on or after this time.` -> `Time corresponding to the end of twilight on or after this time.`